### ISSUE TITLE
The KeyboardManager#hasFocus property now reports whether any focusable HTML element currently has focus. If so we prevent keybinding actions. This improves upon a prior accessibility issue where certain element types like <a> links would trap focus and instead execute keybindings.

### ISSUE THREAD

**Issue by mouse0270** [foundryvtt/foundryvtt#8654]
### User Experience

When tabbing through a window if you tab onto an anchor element you appear to get locked and unable to continue tabbing. For example, you can test this by opening up the module management window and then expanding the modules to see the module details. Then focus on the search field and tab until you hit the first author link and you won't be able to tab any further.

**Comment by mouse0270** [comment]
It appears to have something to do with this key binding, as switching this key binding away from tab fixes this issue.
![image](https://user-images.githubusercontent.com/564874/208260103-5982463f-f3ee-476e-87d4-f2fb14df3d26.png)
